nanoFramework.Json.Test/JsonUnitTests.cs

```diff
@@ -1384,6 +1384,34 @@ public void DeserializeObjectWithStringContainingNonAsciiChars()
             Assert.Equal(input.Value, result.Value);
         }
 
+        [TestMethod]
+        public void DeserializeObjectWithJsonIgnoreAttribute()
+        {
+            OutputHelper.WriteLine("Starting JsonIgnore Test...");
+            Json.Configuration.Settings.UseIgnoreAttribute = true;
+            OutputHelper.WriteLine("UseIgnoreAttribute enabled.");
+
+            var testObject = JsonIgnoreTestClass.CreateTestClass();
+            var jsonString = JsonSerializer.SerializeObject(testObject);
+            OutputHelper.WriteLine("After serialize.");
+            JsonIgnoreTestClass dserResult = JsonConvert.DeserializeObject(jsonString, typeof(JsonIgnoreTestClass)) as JsonIgnoreTestClass;
+            OutputHelper.WriteLine("After deserialize.");
+
+            //test serialize and deserialize
+            bool jsonSuccess = testObject.IsEqual(dserResult);
+            Assert.IsTrue(jsonSuccess);
```

Comment:
note that you can as well add an error message in case it's not successful. So you know where it broke and maybe other values you want to track in the test result.